src/lib/utils/meta_table_manager.cpp

```diff
@@ -10,11 +10,18 @@
 #include "storage/segment_iterables/any_segment_iterable.hpp"
 #include "storage/table.hpp"
 #include "storage/table_column_definition.hpp"
+#include "utils/plugin_manager.hpp"
 
 namespace {
 
 using namespace opossum;  // NOLINT
 
+#ifdef __APPLE__
+#define DYNAMIC_LIBRARY_SUFFIX ".dylib"
```

Comment:
Not a fan of macros. I don't know a way around `ifdef __APPLE__`, but we can use a constexpr for the string.